src/routes/deposit.svelte

```diff
@@ -26,3 +32,20 @@
     {#if id}Selected id: {id}{:else}Select an id by typing it in above.{/if}
   </p>
 </div>
+<div>
+  <BulkLookup
+    db="dipstaging"
+    id="aip"
+    label="Paste your ID's to lookup"
+    on:submit={search} />
+</div>
+<div>
+  {#if id !== undefined}
+    <table>
+      <th>List of ID's</th>
+      <tr>
+        <td class="py-2 px-2">{id}</td>
+      </tr>
+    </table>
+  {/if}
```

Comment:
You probably don't want to be generating a table with one column per ID, which is what you get here.

Reply:
Yeah, This is one of the obvious changes that i was correcting now!  and the other is the filter(as per the previous comment.